src/main/java/com/basho/riak/client/api/commands/kv/CoveragePlan.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2013-2015 Basho Technologies Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.basho.riak.client.api.commands.kv;
+
+import com.basho.riak.client.api.RiakCommand;
+import com.basho.riak.client.api.commands.CoreFutureAdapter;
+import com.basho.riak.client.core.RiakCluster;
+import com.basho.riak.client.core.RiakFuture;
+import com.basho.riak.client.core.operations.CoveragePlanOperation;
+import com.basho.riak.client.core.query.Namespace;
+
+// TODO: documentation for CoveragePlan needs to be improved
```

Comment:
@ph07 @laurenrother as @alexmoore mentions, note that when we add Parallel Extract support to the Riak Java Client we'll need to update docs to say the feature is only supported in TS.